fusestore/store/src/configs/config.rs

```diff
@@ -39,7 +39,7 @@ impl Config {
     }
 
     /// Create configs from args.
-    pub fn create_from_args() -> Self {
+    pub fn load_from_args() -> Self {
         let mut cfg = Config::from_args();
         cfg.version = include_str!(concat!(env!("OUT_DIR"), "/version-info.txt")).to_string();
```

Comment:
By the way, FuseQuery is not use version-info.txt anymore:

https://github.com/datafuselabs/datafuse/blob/master/fusequery/query/src/configs/config.rs#L141-L152

It's using `VERGEN` instead.

Reply:
Sure I see that and am going to fix that part in the next PR